builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1,4 +1,4 @@
-#!groovy.
+#!mgroovy.
```

Comment:
Spell err..

Reply:
Fixed the typo. This really didn't break the build pack.